apps/dashboard/src/@3rdweb-sdk/react/hooks/useEngine.ts

```diff
@@ -145,6 +145,42 @@ export function useEngineSystemHealth(
   );
 }
 
+export interface EngineSystemQueueMetrics {
+  result: {
+    queued: number;
+    pending: number;
+    latency?: {
+      msToSend: { p50: number; p90: number };
+      msToMine: { p50: number; p90: number };
+    };
+  };
+}
+
+export function useEngineSystemQueueMetrics(
+  instanceUrl: string,
+  pollInterval: number | false = false,
+) {
+  const token = useLoggedInUser().user?.jwt ?? null;
+
+  return useQuery(
+    engineKeys.health(instanceUrl),
```

Comment:
this looks like its re-used? every query key HAS to be unique

Reply:
think of these as cache keys, you wouldn't share a redis key with different values in it either

Reply:
also please update to use the object notation `useQuery({queryKey: [], queryFn: () => ...})`

Reply:
Typo on the reusing key name. Will update to object notation.